packages/tasit-sdk/src/testHelpers/helpers.js

```diff
@@ -229,12 +256,79 @@ const duration = {
   },
 };
 
+const etherFaucet = async (provider, fromWallet, beneficiary, amountInWei) => {
+  const connectedFromWallet = fromWallet.connect(provider);
+  const tx = await connectedFromWallet.sendTransaction({
+    // ethers.utils.parseEther("1.0")
+    value: "0x0de0b6b3a7640000",
+    to: beneficiary.address,
+  });
+  await provider.waitForTransaction(tx.hash);
```

Comment:
Yes, because we're bypassing TasitSDK here and using `ethers.js` directly.

Reply:
Yep. What I meant is that it's interesting that we needed to use `ethers.js` directly here

Reply:
Oh, sorry.
Yes. `Account` for now is basically exposing `ethers.js` Wallet object.

Reply:
Could I resolve this one?

Reply:
One more question before we do - why not use `tasit-action` in the Ether faucet instead of ethers.js?

Reply:
`tasit-action` is wrapping the `ethers.Contract` object. `tasit-account` doesn't wrapping `ethers.Wallet` object, because of that we need to use `ethers` to make a transaction from a wallet/account. Account creation from `tasit-action` only return a `ethers.Wallet` object.
This is related to https://github.com/tasitlabs/TasitSDK/issues/118 in some way.

Reply:
Hmm, but wouldn't the standard use case in the app be that `tasit-account` generates an account and then `tasit-action` takes actions on behalf of that account?

This helper is in `tasit-sdk`, so I would think it would be free to use both.

I agree that it's related to that other issue. We can resolve this as long as in the long run you agree that we'd eventually want to use `tasit-action` to do this transaction and not ethers.js

Reply:
Hmmm, got your point now.
I agree that we shouldn't use ethers.js. I'm just couldn't visualize yet how create a wallet transaction using `tasit-action` API.
Since ETH transfer is done by creating, sign and broadcasting a transaction, I was think in something like this:
```javascript
// draft
const account = Account.create();
const transferAction = account.transfer(to, value);
transferAction.send(); // or sendForFree
```

Maybe some code will help me to understand better.

Reply:
Yeah, that draft looks good to me!

Reply:
OK. Extracted to: https://github.com/tasitlabs/TasitSDK/issues/220

Reply:
Thanks! Resolving this